Sources/ToastWindow.swift

```diff
@@ -52,9 +52,12 @@ open class ToastWindow: UIWindow {
     }
     set { /* Do nothing */ }
   }
+  
+  private weak var mainWindow: UIWindow?
 
-  public override init(frame: CGRect) {
+  public init(frame: CGRect, mainWindow: UIWindow?) {
     super.init(frame: frame)
```

Comment:
If you change the method signature and remove the override, then you probably need to call `self.init` instead of `super.init`.

Reply:
Oh, we have to use `self.init` only when use `convenience` init.
However, it's not `convenience` init, so we can use `super.init`.